wavefront_pyformance/wavefront_reporter.py

```diff
@@ -23,7 +24,8 @@ class WavefrontReporter(pyformance.reporters.reporter.Reporter):
 
     # pylint: disable=too-many-arguments
     def __init__(self, source='wavefront-pyformance', registry=None,
-                 reporting_interval=10, clock=None, prefix='', tags=None):
+                 reporting_interval=10, clock=None, prefix='', tags=None,
```

Comment:
default to 60 seconds?

Reply:
@sushantdewan123 Did not touch the interval here. It was as it is.